Readme.md

```diff
@@ -6,53 +6,54 @@ Demo [Orchard Core](https://orchardcore.net/) module for training purposes guidi
 
 If you prefer tutorial videos more then check out the [Dojo Course 3, the complete Orchard Core tutorial series](https://orcharddojo.net/orchard-training/dojo-course-3-the-full-orchard-core-tutorial).
 
-**If you are interested in training materials and personalized Orchard trainings please visit [Orchard Dojo](https://orcharddojo.net/).**
+**If you are interested in training materials and personalized Orchard training please visit [Orchard Dojo](https://orcharddojo.net/).**
```

Comment:
```suggestion
**If you are interested in training material and personalized Orchard training please visit [Orchard Dojo](https://orcharddojo.net/).**
```
I think material is uncountable in the sense it's used here. (as in data and knowledge, as opposed to individual pieces of physical resources)

Reply:
Hmm, looks like it, but the sentence feels weird to me. Let's use "resources".